rune/libenclave/internal/runtime/pal/skeleton/sgxsign.c

```diff
@@ -465,7 +466,10 @@ int main(int argc, char **argv)
 #endif
 	if (enclave_debug)
 		ss.body.attributes |= SGX_ATTR_DEBUG;
-	ss.body.xfrm = 7;
+
+	get_sgx_xfrm_by_cpuid(&xfrm);
```

Comment:
signtool may run on a machine with AVX support, and target machine running enclave may have AVX support. But currently skeleton is just a local sample enclave runtime, assuming the same machine for enclave build and creation.